docker/multiplexer.Dockerfile

```diff
@@ -5,7 +5,7 @@
 # Separating the builder and runtime image allows the runtime image to be
 # considerably smaller because it doesn't need to have Golang installed.
 ARG BUILDER_IMAGE=docker.io/golang:1.24.6-alpine
-ARG RUNTIME_IMAGE=docker.io/alpine:3.19
+ARG RUNTIME_IMAGE=docker.io/alpine:3.20
```

Comment:
[no change needed] looks like latest is 3.22. Any reason to not upgrade to that?

Ref: https://hub.docker.com/_/alpine